presentation-compiler/test/dotty/tools/pc/tests/completion/CompletionArgSuite.scala

```diff
@@ -614,8 +614,9 @@ class CompletionArgSuite extends BaseCompletionSuite:
     check(
       s"""|case class Context()
           |
-          |def foo(arg1: (Context) ?=> Int, arg2: Int): String = ???
-          |val m = foo(ar@@)
+          |object Main:
```

Comment:
We're reusing the presentation compiler, and we still have a problem with mutating context (I may have a fix for this in the future). We should always wrap entries into something, as this will override all previous state.